index.html

```diff
@@ -157,33 +165,134 @@ <h2>
+          <p class="about-me-1">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <div class="green-btn">
+            <a href="index.html">
+              <button type="button" title="press">Get my resume</button>
+            </a>
+          </div>
         </div>
-        <p class="poppage-3">
-          Lorem Ipsum is simply dummy text of the printing and typesetting
-          industry. Lorem Ipsum has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a galley of type and
-          scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing
-          and typesetting industry. Lorem Ipsum has been the industry's standard
-          dummy text ever since the 1500s, when an unknown printer took a galley
-          of type and scrambled it 1960s.
-        </p>
-        <div class="poppage-4">
-          <a href="index.html"
-            ><button title="live" type="button">
-              See Live
-              <img src="assests/images/see live icon.svg" alt="live" /></button
-          ></a>
-          <a href="https://github.com/GeekyHacks/Personal-Portfolio-Microverse">
-            <button title="git" type="button">
-              See Source <img src="assests/images/git.svg" alt="git" /></button
-          ></a>
+        <div class="languages">
+          <img
+            class="languages-img"
+            src="assests/images/icon-languages.svg"
+            alt="languages"
+          />
+
+          <h3 class="languages-text">Languages</h3>
+          <ul class="languages-lis">
+            <li class="small-li" for="html">HTML</li>
+            <li class="small-li" for="JS">JS</li>
+            <li class="small-li" for="CSS">CSS</li>
+            <li class="small-li" for="Ruby">Ruby</li>
+          </ul>
         </div>
+
+        <div class="framworks">
+          <img
+            class="framworks-img"
+            src="assests/images/framworks.svg"
+            alt="framworks"
+          />
+
+          <h3 class="framworks-text">Framworks</h3>
+          <ul class="framworks-lis">
+            <li class="mid-li" for="React.js">React.js</li>
+            <li class="big-li" for="Ruby">Ruby and Rails</li>
+            <li class="small-li" for="rspec">RSpec</li>
+            <li class="mid-li" for="Capybara">Capybara</li>
+            <li class="mid-li" for="Selenium">Selenium</li>
+          </ul>
+        </div>
+        <div class="skills">
+          <img
+            class="skills-img"
+            src="assests/images/skills.svg"
+            alt="skills"
+          />
+
+          <h3 class="skills-text">Skills</h3>
+          <ul class="skills-lis">
+            <li class="big-li" for="Database Management">
+              Database Management
```

Comment:
- [ ] As suggested by the previous reviewer and the project requirement, kindly make a nested ul tag so that it would be semantically correct.

Just as:
```
<ul><!-- start of  main list-->
  <li>HTML</li>
   <li>CSS</li>
    <li>JavaScript
            <ul><!-- start of nested list-->
                <li>Angular</li>
                <li>React</li>
                <li>Vue</li>
            </ul><!--end of nested list-->
     </li>
 </ul><!--end of  main list -->
```